packages/hierarchies/src/test/internal/operators/grouping/PropertiesGrouping.test.ts

```diff
@@ -873,6 +873,55 @@ describe("PropertiesGrouping", () => {
         });
       });
 
+      it("groups node, when property value is navigation", async () => {
+        const nodes = [
+          createTestProcessedInstanceNode({
+            key: { type: "instances", instanceKeys: [{ className: "TestSchema.Class", id: "0x1" }] },
+            parentKeys: ["x"],
+            processingParams: {
+              grouping: {
+                byProperties: {
+                  propertiesClassName: "TestSchema.Class",
+                  createGroupForUnspecifiedValues: true,
+                  propertyGroups: [{ propertyName: "PropertyName" }],
+                },
+              },
+            },
+          }),
+        ];
+        const property = { name: "PropertyName", isPrimitive: () => false, isNavigation: () => true } as unknown as EC.Property;
+        const stubbedClass = imodelAccess.stubEntityClass({
+          schemaName: "TestSchema",
+          className: "Class",
+          is: async () => true,
+          properties: [property],
+        });
+        const propertyInfo: propertiesGrouping.PropertyGroupInfo = {
+          ecClass: stubbedClass,
+          previousPropertiesGroupingInfo: [],
+          propertyGroup: { propertyName: "PropertyName" },
+        };
+        const expectedGroupingNodeKey: GroupingNodeKey = {
+          type: "property-grouping:value",
+          propertyName: "PropertyName",
+          propertyClassName: "TestSchema.Class",
+          formattedPropertyValue: "",
```

Comment:
IMO, maybe the test should use the general case when there is a valid property value

Reply:
Changed the propertyValue to a `ConcatenatedValue`